src/main/java/mil/dds/anet/beans/Position.java

```diff
@@ -233,6 +246,11 @@ public void setPreviousPeople(List<PersonPositionHistory> previousPeople) {
     this.previousPeople = previousPeople;
   }
 
+  @GraphQLInputField(name = "allPreviousPeople")
+  public void setAllPreviousPeople(List<PersonPositionHistory> previousPeople) {
+    this.previousPeople = previousPeople;
+  }
+
```

Comment:
This field should not be settable through GraphQL (query-only field, if it should be there at all).

Reply:
GraphQLInput removed.